src/modules/campaign-monitor/templates/templates.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Template } from '../campaign-monitor.types';
+
+export class TemplateService {
```

Comment:
REQ: You can just do similar logic as you dit with xml & csv. No need for Class.